service/service.go

```diff
@@ -276,6 +283,36 @@ func setLogFormatAndLevel(logFormat log.Formatter, level log.Level) {
 	log.SetLevel(level)
 }
 
+func getStorageArrays(secretParams *viper.Viper, opts *Opts) () {
+		// Access the storagearrays key (which is a slice of maps)
+		storageArrays := secretParams.Get("storagearrays").([]interface{})
+		if storageArrays == nil {
+			log.Println("No storage array declared.")
+		} else {
+			// Ensure there's at least one server and extract labels and parameters if any
+			if len(storageArrays) == 0 {
+				log.Println("No storage arrays found.")
+			} else {
```

Comment:
You could simplify this block and remove the nested if else by checking len(storageArrays). The len() function is tolerant of nil as a value.

Reply:
done.